dataland-frontend/tests/e2e/plugins/index.js

```diff
@@ -3,22 +3,25 @@ module.exports = (on, config) => {
   config.fixturesFolder = "../testing/data";
   config.supportFile = "tests/e2e/support/index.ts";
   config.env.commit_id = require("git-commit-id")({ cwd: "../" });
-  switch (process.env.ENVIRONMENT) {
-    case "development": {
+  switch (process.env.REALDATA) {
+    case "false": {
```

Comment:
In unserem SetEnv Script wird die Env-Variable "REALDATA" auf false gesetzt, was dafür sorgt, dass bei mir jetzt nur das runAll.ts Script angezeigt wird. Eigentlich sollten beim lokalen Entwickeln aber alle Tests dargstellt werden. (PS: Alternativ schreibe ich diese Datei und das Management welcher Test wann ausgeführt wird gerad für das Test-Refactoring sowieso anders)

Reply:
Wenn REALDATA ungesetzt ist, ist alles normal.
Habe das also aus dem setEnv Script rausgenommen. 

Was meinst du - hast du eine bessere Idee? Oder taugt es bis du ohnehin alles anders machst?

Reply:
Es tuts auf jeden Fall, bis ich das wieder anders mache